oaknational/Oak-Web-Application
# fix: trim short answer text to avoid case and ws comparison

## Description

Music year: 1955

- List of changes

## Issue(s)

There are instances in the CAT where Short answer question answers have been added with a trailing space at the end e.g. https://creator-staging.thenational.academy/app/quiz-builder?=#lesson_id=1768&programme_id=30&quiz_type=exit&unit_id=157&unitvariant_id=159

When a user enters the correct answer but without a space they are marked as incorrect

## How to test

1. Go to https://deploy-preview-2936--oak-web-application.netlify.thenational.academy/pupils/lessons/the-wall-street-crash/starter-quiz
2. Enter "  Tariff  " as the answer
3. observe it is marked correct
4. try other incorrect answers and observe they are marked as incorrect

## Checklist

- [x] white space is ignored when grading short answers
- [x] capitalisation is ignored when grading short answers
- [x] other grading is uneffected
